articles/unique_network_flows.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: default
+title: Unique Network Flows
+permalink: articles/unique_network_flows.html
+abstract: Enable unique network flow identifiers for publishers and subscriptions in communicating nodes
+author: '[Ananya Muddukrishna, Ericsson AB](https://github.com/anamud)'
+published: true
+---
+
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+# Unique Network Flows
+
+For performance, ROS2 applications require careful selection of QoS for publishers and subscriptions. Although networks offer various QoS options, ROS2 publishers and subscriptions are unable to use them due to non-unique flows. As a result, ROS2 publishers and subscriptions can only hope to obtain undifferentiated QoS from networks. This ultimately degrades the performance potential of ROS2 applications and wastes networking infrastructure investments.
+
+We propose unique network flows for ROS2 publishers and subscriptions. Our proposal is easy to use, convenient to implement, and minimal. Plus, it respects non-DDS-middleware-friendly concerns in ROS2.
+
+In this document, we first describe essential background concepts. After that we precisely state the problem and propose a solution to the problem. Existing solutions are compared in the end.
+
+## Background
+
+IP networking [1] is the pre-dominant inter-networking technology used today. Ethernet, WiFi, 4G/5G telecommunication all rely on IP networking.
```

Comment:
@anamud nit: 
```suggestion
IP networking [1] is the predominant inter-networking technology used today. Ethernet, WiFi, 4G/5G telecommunication all rely on IP networking.
```
? It's the first time I've predominant written this way, so maybe I'm just not familiar with this particularly spelling.

Reply:
Its a typo. Thanks for spotting it. I have a nasty habit of over-using hyphens.

Reply:
Fixed typo in separate [commit](https://github.com/ros2/design/pull/304/commits/37d444786932c06be71a54f5d8daad0806bd9756) below.